### PR TITLE
USHIFT-915: Add wait flag when creating VMs to make sure they are restarted

### PR DESCRIPTION
**Pull request by ggiguash**
Closes [USHIFT-915](https://issues.redhat.com//browse/USHIFT-915)

### CODE REVIEW

File: docs/devenv_setup.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ virt-install \
     --os-type generic \
     --events on_reboot=restart \
     --cdrom ./${ISONAME} \
+    --wait \
```
Comment (dhensel-rh) [inline docs/devenv_setup.md:35]: Would it be ok to add a time value (in minutes) to this ? I do not know what the default is if you do not include a time value. I have been using --wait 5, and it works ok. Thoughts ?

File: docs/rhel4edge_iso.md [hunk 2]
```diff
@@ -168,6 +168,7 @@ virt-install \
     --os-type generic \
     --events on_reboot=restart \
     --cdrom ./microshift-installer-*.$(uname -i).iso \
+    --wait \
```
Comment (dhensel-rh) [inline docs/rhel4edge_iso.md:171]: Would it be ok to add a time value (in minutes) to this ? I do not know what the default is if you do not include a time value. I have been using --wait 5, and it works ok. Thoughts ?

Comment (ggiguash) [reply docs/rhel4edge_iso.md:171]: The manual states:
```
--wait [WAIT]         Minutes to wait for install to complete.
```
So, the wait timeout is optional. I think we should just wait indefinitely until the process completes or fails. 
Picking the "right" timeout is always tricky and I'm not sure it's necessary in this case.

File: docs/devenv_setup.md (hunk 1 above)
Comment (ggiguash) [reply docs/devenv_setup.md:35]: I responded below.

**Pull request merged**